spec/features/admin_spec.rb

```diff
@@ -56,19 +56,40 @@
       expect(page).to have_selector('#users-information', visible: false)
       expect(page).to have_selector('#practice-leaderboards', visible: true)
       expect(page).to have_selector('#practice-search-terms-table', visible: false)
+      expect(page).to have_css("input[value='Export as .xlsx']", visible: false)
 
       click_link('Practice Search Terms')
       expect(page).to have_selector('#users-information', visible: false)
       expect(page).to have_selector('#practice-leaderboards', visible: false)
       expect(page).to have_selector('#practice-search-terms-table', visible: true)
+      expect(page).to have_css("input[value='Export as .xlsx']", visible: false)
 
       click_link('Users Information')
       expect(page).to have_selector('#users-information', visible: true)
       expect(page).to have_selector('#practice-leaderboards', visible: false)
       expect(page).to have_selector('#practice-search-terms-table', visible: false)
+      expect(page).to have_css("input[value='Export as .xlsx']", visible: false)
+
+      click_link('Metrics')
+      expect(page).to have_selector('#users-information', visible: false)
+      expect(page).to have_selector('#practice-leaderboards', visible: false)
+      expect(page).to have_selector('#practice-search-terms-table', visible: false)
+      expect(page).to have_css("input[value='Export as .xlsx']", visible: true)
     end
   end
 
+  it 'should show allow admin to download metrics as .xlsx file' do
+    login_as(@admin, scope: :user, run_callbacks: false)
+    visit '/admin'
+
+    click_link('Metrics')
+    export_button = find(:css, "input[type='submit']")
+    export_button.click
```

Comment:
⚠️ I believe this downloads a file locally whenever we run this... Not sure if we want to do that... 🤔

Reply:
hmmm can we clear the file after the test?
`FileUtils.rm_rf(<file here>)`

Reply:
yay thank you for the guidance :)